developers/dev-faq.md

```diff
@@ -0,0 +1,82 @@
+</details>
+
+<details>
+<summary>Is there a trading platform or stock exchange that has successfully forked the Ocean marketplace codebase?</summary>
+
+Ocean technology is actively used by Daimler/Acentrik, deltaDAO/GAIA-X, and several other entities. You can find further details on the Ocean [ecosystem page](https://oceanprotocol.com/explore/ecosystem).
+
+</details>
+
+<details>
+<summary>What are the Ocean faucets and how can they be used?</summary>
+
+An Ocean faucet is a site to get (fake) OCEAN for use on a given testnet. There's an Ocean faucet for each testnet that Ocean is deployed to. The [networks](../discover/networks/) page have more information.
+</details>
+
+<details>
+<summary>Can I use the off-the-shelf CSS available in the repo?</summary>
+
+The marketplace name, logo, and typeface must be changed by the client. A slight modification would be enough for compliance. For more information consult the README file about [forking Ocean Market](https://github.com/oceanprotocol/market#-forking). 
+</details>
+
+<details>
+<summary>How can I convert tokens from the BEP20 network to the ERC20 network?</summary>
+
+Please follow this [tutorial](../discover/networks/bridges#binance-smart-chain-bsc-bridge) to bridge from/to binance smart chain. Please double-check the addresses and make sure you are using the right smart contracts.
+</details>
+
+<details>
+<summary>How to bridge my mOcean back to Ocean?</summary>
+
+Please follow this [tutorial](../discover/networks/bridges#polygon-ex-matic-bridge) to bridge to/from polygon. Please double-check the addresses and make sure you are using the right smart contracts.
+</details>
+
+<details>
+<summary>Is it possible to reverse engineer a dataset on Ocean by having access to both the algorithm and the output? </summary>
+
+Not to our knowledge. But please, give it a shot and share the results with us 😄
+
+PS: We offer good rewards 😇
+</details>
+
+<details>
+<summary>If a dataset consists of 100 individuals' private data, does this solution allow each individual to maintain sovereign control over their data while still enabling algorithms to compute as if it were one dataset?</summary>
+
+Each individual could publish their dataset themselves, to get a data NFT. From the data NFT, they can mint datatokens which are to access the data. They have sovereign control over this, as hold the keys to the data NFTs and datatokens, and have great flexibility in how to give others access. For example, they could send a datatoken to a DAO for the DAO can manage. Or they could grant datatoken-minting permissions to the DAO. The DAO could use this to assemble a dataset across 100 individuals. ⁣
```

Comment:
Add "Yes. " to the beginning of this paragraph.

Why: to answer the question directly!